ResearchKit/Common/ORKScaleSlider.m

```diff
@@ -96,10 +164,9 @@ - (void)updateValueForTouchAtPoint:(CGPoint)touchPoint {
 }
 
 static CGFloat kLineWidth = 1.0;
-- (void)drawRect:(CGRect)rect
-{
-    CGRect trackRect = [self trackRectForBounds:[self bounds]];
-    CGFloat centerY = [self bounds].size.height / 2.0;
+- (void)drawRect:(CGRect)rect {
+    CGRect trackRect = [self trackRectForBounds:self.bounds];
+    CGFloat centerY = self.bounds.size.height / 2.0;
```

Comment:
This is ok, but we do prefer the bounds property calls to be written as method calls to make the fact that they're method calls explicit. If I were going to make a change here, for instance, I'd factor out the call to the bounds method and then re-use the struct for both the following calculations.